src/Detached.Mappers/TypeMappers/Entity/Complex/ComposedEntityTypeMapper.cs

```diff
@@ -54,7 +54,9 @@ public override TTarget Map(TSource source, TTarget target, IMapContext context)
                 else
                 {
                     TKey targetKey = _getTargetKey(target, context);
-                    if (Equals(sourceKey, targetKey))
+                    bool isOwnedEntityWithoutKey = targetKey is NoKey && sourceKey is NoKey;
```

Comment:
1 - I'm concerned about this change, could it break other things?
2 - Shall we make NoKey == NoKey?? (As NoKey is our own object) that would be a lot faster

Reply:
The other way would be to write your own type mapper for owned entities. For this an AnnotationHandler that points to the Owned Attributes. I tried it, but can't include the Nuget Package from EF Core.

Reply:
I can implement this feature, but EF Core is not .Net Standard Compatible. I can not install the Package with the Owned Attribute.
I tried it with entityType.isOwned() but isOwned() applies to more than just Owned Entities. Because internally ef core looks with isOwned() if it has a ForeignKey.
I'm going to build it in now and put a PR....
It is your decision.